spring-cloud-gcp-samples/spring-cloud-gcp-integration-storage-sample/src/main/resources/application.properties

```diff
@@ -2,4 +2,4 @@
 
 gcs-read-bucket=[YOUR_BUCKET_NAME]
 gcs-write-bucket=[YOUR_WRITE_BUCKET_NAME]
-local-directory=[YOUR_LOCAL_FILE_SYSTEM_TEMP_FILE_FOLDER]
+gcs-local-directory=[YOUR_LOCAL_FILE_SYSTEM_TEMP_FILE_FOLDER]
```

Comment:
I'm not sure about this rename, but in any case, you should also update it in the `README.adoc`

Reply:
Done. I am open to other suggestions on this - I renamed it since once the properties were set in the travis.yml, just having the property "local-directory" seemed too general and adding a "gcs-" prefix would give context on what the property is used for in the yml file.